rules-reviewed/eap7/eap6/tests/eap6-xml.windup.test.xml

```diff
@@ -6,11 +6,13 @@
             <rule id="eap6-xml-01000-test">
                 <when>
                     <not>
-                        <hint-exists message="CMP entity beans are no longer supported in JBoss EAP 7. User is requested to use JPA"/>
+                      <iterable-filter size="2">
+                        <hint-exists message="Configuration provided in this ejb-jar.xml should be configured using JPA persistence.xml or using JPA annotations."/>
+                      </iterable-filter>
                     </not>
                 </when>
                 <perform>
-                    <fail message="CMP Entity EJB hint not found!"/>
+                    <fail message="There should be 2 hints for CMP Entity definitions in ejb-jar.xml"/>
```

Comment:
the message was equal for hints that are registered in JavaClass so our test didn't find out there is a bug.